### PR TITLE
errors, casing, formatting

### PR DESCRIPTION
**Pull request by drewmalin**
First pass at error handling, casing, spelling, formatting, etc.

No new functionality introduced (other than more direct failure on certain errors seen by cobra.

### CODE REVIEW

File: internal/auth/login.go [hunk 1]
```diff
@@ -62,29 +63,33 @@ var successHTML embed.FS
...
 		return err
 	}
 
 	// TODO: pretty print URL?
-	fmt.Println(cotterURL)
+	fmt.Fprintln(context.Out, cotterURL)
 
 	err = openInDefaultBrowser(cotterURL)
 	if err != nil {
+		context.PrintErr("Failed to open default browser", err)
```
Comment (naderkhalil) [inline internal/auth/login.go:80]: Where does context printing go?

File: internal/cmdcontext/cmdcontext.go [hunk 2]
```diff
@@ -39,3 +40,8 @@ func (c *Context) Init(verbose bool) {
 	c.VerboseOut = os.Stdout
 	c.Err = os.Stderr
 }
+
```
Comment (naderkhalil) [inline internal/cmdcontext/cmdcontext.go:43]: Is this so the printing is consolidated?

File: internal/endpoint/endpoint.go [hunk 3]
```diff
@@ -23,57 +23,77 @@ import (
...
+	if err != nil {
+		context.PrintErr("Failed to get active project", err)
+		return err
+	}
 
-	token, _ := auth.GetToken()
-	brevAgent := brev.BrevAgent{
+	token, err := auth.GetToken()
+	if err != nil {
+		context.PrintErr("Failed to retrieve auth token", err)
```
Comment (naderkhalil) [inline internal/endpoint/endpoint.go:41]: Is the pattern that GetToken just returns error, and that can be tossed through the call stack until the logic layer decides to print?

File: internal/version/version.go [hunk 4]
```diff
@@ -2,6 +2,7 @@ package version
 
 import (
 	"fmt"
+
```
Comment (naderkhalil) [inline internal/version/version.go:5]: is convention for our packages vs system packages to be separated by new line?

**Review by naderkhalil — APPROVED** [review]

**Comment by naderkhalil** [comment]
I'm getting a seg fault if i run this command:

`./../bin/osx/brev endpoint remove --name ep2`

File: internal/auth/login.go (hunk 1 above)
Comment (drewmalin) [reply internal/auth/login.go:80]: Admittedly "context" is probably too vague a word, but this is a utility method that allows for the '--verbose' flag to work with error messages, and for the error content itself to properly make its way to stderr.

File: internal/cmdcontext/cmdcontext.go (hunk 2 above)
Comment (drewmalin) [reply internal/cmdcontext/cmdcontext.go:43]: Exactly!

File: internal/version/version.go (hunk 4 above)
Comment (drewmalin) [reply internal/version/version.go:5]: Yeah the way I like doing it is:

```go
import(
  <standard library>
  <standard library>
  <standard library>
  
  <third party library>
  <third party library>
  <third party library>

  <local package>
  <local package>
  <local package>
)

File: internal/endpoint/endpoint.go (hunk 3 above)
Comment (drewmalin) [reply internal/endpoint/endpoint.go:41]: Yes -- with a lot of these "lower level" packages I think we should have their errors not necessarily assume that they are running within the context of a CLI. So they should ideally just return the error (one which is fully formed and has a useful message, that is) and let the caller deal with it (ignore, warn, log, print, etc.)

**Review by naderkhalil — APPROVED** [review]

**Pull request merged**